dev-docs/bidders/publicgood.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: bidder
+title: Public Good
+description: Public Gppd Bid Adapter
+biddercode: publicgood
+pbjs: true
+sidebarType: 1
+media_types: banner
+safeframes_ok: true
+tcfeu_supported: false
+usp_supported: false
+coppa_supported: false
```

Comment:
There's quite some meta data missing. Please refer to https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter and add as much as possible.

media_types is especially relevant.